core/src/sequences.ts

```diff
@@ -74,6 +74,10 @@ export class Sequences {
     return n && (n & (n - 1)) === 0;
   }
 
+  public static clone(ns:INoteSequence) {
```

Comment:
Thanks, makes sense to centralize this logic. Could you also update sequences_test.ts to use this in the few places it clones NoteSequences?